azure-iot-device/azure/iot/device/provisioning/provisioning_device_client_factory.py

```diff
@@ -0,0 +1,34 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
>-------------------------------------------------------------------------- [](start = 2, length = 74)

I assume all the files added under azure/iot/device/provisioning are merge mistakes as well as the files added under test/provisoning

Reply:
Yup. Thanks.

---
In reply to: [290980777](https://github.com/Azure/azure-iot-sdk-python-preview/pull/101#discussion_r290980777) [](ancestors = 290980777)